tests/unittest/test_breakpad_resource.py

```diff
@@ -346,13 +374,14 @@ def test_extract_payload_invalid_json_not_dict(self, request_generator):
         with pytest.raises(MalformedCrashReport, match="invalid_json_value"):
             bsp.extract_payload(req)
 
-    def text_extract_payload_kvpairs_and_json(self, request_generator, metricsmock):
-        # If there's a JSON blob and also kv pairs, then that's a malformed
-        # crash
+    def test_extract_payload_kvpairs_and_json(self, request_generator, metricsmock):
```

Comment:
This test never ran because it started with `text`. This tests what happens if a crash report includes annotations in multipart format and annotations in the `extra` field in a JSON-encoded value.

Once I fixed the test method name, it ran and failed because it didn't account for the change in how we handle this situation. I updated this test.